src/main/java/cs/ualberta/cmput402/tictactoe/board/Board.java

```diff
@@ -128,5 +130,13 @@ public Player getPlayerAtPos(int row, int col){
         return board[row][col];
     }
 
-
+    private boolean boardIsFull(){
```

Comment:
It is a good idea to have documentation for functions.

Reply:
This is addressed in commit: abc14b9